projects/storefrontlib/cms-components/misc/icon/icon.component.html

```diff
@@ -1 +1,3 @@
-<i [outerHTML]="icon"></i><ng-content></ng-content>
+<ng-container *ngIf="isTextIcon()">{{ icon }}</ng-container
+><svg *ngIf="isSvgIcon()"><use [attr.xlink:href]="icon"></use></svg
+><ng-content></ng-content>
```

Comment:
Formatting is not nice, because there is no space allowed between either icon type and the projected content. This mimics the behavior of the old implementation. If whitespace is ok after all, we can form at this a lot nicer.